src/chapters/introduction/best-practices.rst

```diff
@@ -1,33 +1,37 @@
-Best Practices: Learning To Code
-=================================
+Tips and Motivation for Learning To Code
```

Comment:
These are all good changes for this page.  I agree with them as-is except for one small hiccup in the Social Media section.

Reply:
Addressed!